semantic/src/site/globals/site.overrides

```diff
@@ -531,7 +551,7 @@ a:hover {
 
 #list-container {
   padding: 25px 0;
-  max-width: @applicationListWidth;
+  width: @applicationListWidth;
```

Comment:
Maybe this is making the table looks bad in some resolutions... is cutting off: from 770px lenght - not a bug deal though

Reply:
The table always went really ugly at ~750px, so I set a minimum to that until we can improve mobile table display properly. So it'll spill off the screen, but at least it won't go completely unreadable like it does if you let it go smaller than 750.